src/PABLO/LocalTree.hpp

```diff
@@ -211,12 +211,9 @@ class LocalTree{
     void        findEdgeNeighbours(const Octant* oct, uint8_t iedge, u32vector & neighbours, bvector & isghost, bool onlyinternal) const;
     void        findNodeNeighbours(const Octant* oct, uint8_t inode, u32vector & neighbours, bvector & isghost, bool onlyinternal) const;
 
-    BITPIT_DEPRECATED(void 		findPeriodicNeighbours(const Octant* oct, uint8_t iface, u32vector & neighbours, bvector & isghost) const);
 	void 		findGhostNeighbours(uint32_t idx, uint8_t iface, u32vector & neighbours, bvector & isghost) const;
 	void 		findGhostEdgeNeighbours(uint32_t idx, uint8_t iedge, u32vector & neighbours, bvector & isghost) const;
 	void 		findGhostNodeNeighbours(uint32_t idx, uint8_t inode, u32vector & neighbours, bvector & isghost) const;
-	BITPIT_DEPRECATED(void 		findGhostPeriodicNeighbours(const Octant* oct, uint8_t iface, u32vector & neighbours) const);
-
```

Comment:
The commit title is "PABLO: added node and edge periodic neighbors", but you are also removing two functions. Better remove this two functions in a separate commit.

Reply:
As above...